src/main.cpp

```diff
@@ -45,5 +50,30 @@ void mainLoop(void)
     // TODO: link module exited, attempt recovery
   }
 
-  // TODO: Failure actions (alarm, watchdog)
+  // Scan through all alarms, looking for any set warning that need to be addressed
+  struct alarmProperties properties = {0};
```

Comment:
do `preventWatchdog` and `suppressionTimeout` not need initialized values?

Reply:
`{0}` will zero initialize the entire structure, which is fine for everything we want to use this structure for (getting the properties of the highest priority alarm and whether or not any alarms are triggered that will block the watchdog)